cmake/CMakeLists.txt

```diff
@@ -652,10 +629,13 @@ endif()
 #Need python to generate def file
 if(onnxruntime_BUILD_SHARED_LIB OR onnxruntime_ENABLE_PYTHON)
   if(onnxruntime_ENABLE_PYTHON)
-    find_package(PythonInterp 3.5 REQUIRED)
-    find_package(PythonLibs 3.5 REQUIRED)
+    if(onnxruntime_ENABLE_LANGUAGE_INTEROP_OPS)
+      find_package(Python 3.5 COMPONENTS Interpreter Development NumPy)
```

Comment:
>3.5 [](http://example.com/codeflow?start=26&length=3)

is 3.6 the minimum Python version?
e.g., [https://github.com/microsoft/onnxruntime/blob/c45ac166d36b28691962e215a3ca30343156d44b/tools/ci_build/build.py#L54](https://github.com/microsoft/onnxruntime/blob/c45ac166d36b28691962e215a3ca30343156d44b/tools/ci_build/build.py#L54)
(and the comment for that function is outdated)